pandas/tests/window/test_numba.py

```diff
@@ -207,8 +220,12 @@ def test_cython_vs_numba_times(self, grouper, nogil, parallel, nopython, ignore_
         )
 
         engine_kwargs = {"nogil": nogil, "parallel": parallel, "nopython": nopython}
-        result = ewm.mean(engine="numba", engine_kwargs=engine_kwargs)
-        expected = ewm.mean(engine="cython")
+
+        # TODO: why only in these cases?
```

Comment:
Because column "A" is string dtype and in the non-group case there's an attempt to agg over the column, while in the groupby case, the grouping column should be dropped internally before hand